_includes/master/charts/calico/values.yaml

```diff
@@ -1,7 +1,4 @@
-# Select which datastore mode. Can be 'etcd' or 'kdd'.
```

Comment:
I think we want to delete this file now so its not checked into the repository.

The helm ruby plugin doesn't need this any more since it is building a tmp values.yaml file for each invocation.